chain/epoch-manager/src/lib.rs

```diff
@@ -111,6 +111,14 @@ impl EpochInfoProvider for EpochManagerHandle {
         let epoch_manager = self.read();
         epoch_manager.minimum_stake(prev_block_hash)
     }
+
+    fn account_id_to_shard_id(
+        &self,
+        account_id: &AccountId,
+        epoch_id: &EpochId,
+    ) -> Result<ShardId, EpochError> {
+        EpochManagerAdapter::account_id_to_shard_id(self, account_id, epoch_id)
+    }
```

Comment:
I'm a bit surprised that you needed to add it, wasn't there one already there?

Reply:
No, not on that specific trait.